src/Tizen.Applications.WidgetControl/Tizen.Applications/WidgetControl.cs

```diff
@@ -435,6 +435,34 @@ public bool IsNoDisplay
             }
         }
 
+        /// <summary>
+        /// Gets setup app ID of the widget.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
```

Comment:
Current API version is API8. Current Platform version is tizen_6.0. It should be:
```suggestion
        /// <since_tizen> 8 </since_tizen>
```